python/cuml/cuml/internals/logger.pyx

```diff
@@ -2,11 +2,6 @@
 # SPDX-FileCopyrightText: Copyright (c) 2020-2025, NVIDIA CORPORATION.
 # SPDX-License-Identifier: Apache-2.0
 #
-
-# distutils: language = c++
-
-
-import logging
```

Comment:
Changes in this file are just deleting some previously dead code and a few style lints.